src/components/Books.js

```diff
@@ -1,15 +1,20 @@
-import { useState } from 'react';
+import { useEffect, useState } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
 import { v4 as uuid } from 'uuid';
 
 import Book from './Book';
 import AddForm from './AddForm';
-import { addBook, deleteBook } from '../redux/books/booksSlice';
+import { fetchBooks, addBook, deleteBook } from '../redux/books/booksSlice';
 
 function BookList() {
-  const books = useSelector((state) => state.books);
+  const books = useSelector((state) => state.books.books);
+  // console.log('Books:', books);
```

Comment:
- [ ] Great job working on this milestone. Kindly remove commented lines of codes from the codebase. This makes the codebase look clean and professional.

Reply:
It's done